package.json

```diff
@@ -34,12 +35,15 @@
     "eslint-plugin-react": "^7.22.0",
     "eslint-plugin-react-hooks": "^4.2.0",
     "eslint-plugin-sonarjs": "^0.6.0",
+    "husky": "^5.1.3",
+    "lint-staged": "^10.5.4",
```

Comment:
Are we still ditching lint staged for pr level linting

Reply:
oh great catch! Just set it up 🤩 !